config/index.js

```diff
@@ -28,6 +29,7 @@ const CONTRACTS = IS_MAINNET
       USDT: '0xde3A24028580884448a5397872046a019649b084'
     }
   } : {
+    GOVERNANCE: '0x914556b16c1220e4af63084dB1acbD4e6f9c65Aa',
```

Comment:
Fuji Governance contract is not the same address as Mainnet Governance contract

Reply:
I will update it on next PR